packages/lab/src/drawer/Drawer.tsx

```diff
@@ -71,58 +62,53 @@ export const Drawer = forwardRef<HTMLDivElement, DrawerProps>(function Drawer(
 
   const [isAnimating, setIsAnimating] = useState(false);
```

Comment:
More of a general question, setIsAnimating is only set to true in the code and will never go back to false. Is this an issue?

Reply:
I was refactoring this and just realised it's not even needed anymore (I think it was a leftover from the `disableAnimations` prop we removed) so I've removed it